packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -17,6 +16,10 @@ import 'colors.dart';
 // This value was eyeballed from a physical device running iOS 13.1.2.
 const double _kOpenScale = 1.1;
 
+// The duration it takes for the CupertinoContextMenu to open.
+// This value was eyeballed from the XCode simulator running iOS 16.0
+const Duration _previewLongPressTimeout = Duration(milliseconds: 900);
```

Comment:
This new constant and the comment explaining where it came from look great to me 👍